### PR TITLE
Add CircleCI log collection to CircleCI CI Visibility docs

### PR DESCRIPTION
**Pull request by calvarjorge**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/pipelines/circleci.md [hunk 1]
```diff
@@ -100,3 +108,5 @@ The [Pipelines][4] and [Pipeline Executions][5] pages populate with data after t
 [7]: /continuous_integration/pipelines/custom_tags_and_metrics/?tab=linux
 [8]: /account_management/teams/
 [9]: https://raw.githubusercontent.com/DataDog/ci-visibility-circle-ci/main/service_hooks.py
+[10]: /continuous_integration/pipelines/circleci/#enable-log-collection
+[11]: integrations/circleci/#setup
```
Comment (brett0000FF) [inline content/en/continuous_integration/pipelines/circleci.md:112]:
```suggestion
[11]: /integrations/circleci/#setup
```

File: content/en/continuous_integration/pipelines/circleci.md [hunk 2]
```diff
@@ -80,6 +82,12 @@ To set custom tags to all the pipeline and job spans generated by the integratio
 #### Integrate with Datadog Teams
 To display and filter the teams associated with your pipelines, add `team:<your-team>` as a custom tag. The custom tag name must match your [Datadog Teams][8] team handle exactly.
 
+### Enable log collection
+
+The DataDog CircleCI integration allows to collect logs from your finished CircleCI jobs and forward them to DataDog.
```
Comment (brett0000FF) [inline content/en/continuous_integration/pipelines/circleci.md:87]:
```suggestion
The Datadog CircleCI integration collects logs from your finished CircleCI jobs and forwards them to Datadog.
```

File: content/en/continuous_integration/pipelines/circleci.md [hunk 3]
```diff
@@ -80,6 +82,12 @@ To set custom tags to all the pipeline and job spans generated by the integratio
...
+
+To enable this feature, follow [this guide][11] to install and configure the DataDog CircleCI integration.
```
Comment (brett0000FF) [inline content/en/continuous_integration/pipelines/circleci.md:89]:
```suggestion
To install and configure this integration, follow the [CircleCI setup guide][11].
```

**Review by brett0000FF — CHANGES_REQUESTED** [review]
Some language tweak recommendations and fixing the broken link. Let me know if you have any questions. 🙂

**Review by drodriguezhdez — APPROVED** [review]

**Review by brett0000FF — APPROVED** [review]
Thanks!

**Pull request merged**